gsoc/gsoc2016/ideas_eras.md

```diff
@@ -0,0 +1,44 @@
+# Italian Mars Society's ERAS Project #
+
+## Sub-Org Details ##
+
+Sub-org admin: Antonio Del Mastro - <gsoc@marssociety.it>
+
+Main repository: <https://bitbucket.org/italianmarssociety/eras/src>
+
+Mailing list: <https://groups.google.com/forum/#!forum/erasproject>
+
+
+## Ideas ##
+
+### 1 – SOLAR STORMS FORECASTING SERVER ###
+
+DIFFICULTY LEVEL: INTERMEDIATE / HARD
+
+MENTORS: [@aldebran](https://bitbucket.org/aldebran/) , [@OPersian](https://bitbucket.org/OPersian/)
```

Comment:
Could you follow the same format that the rest of the pages? Italics and full names on the mentors?